xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -265,4 +266,34 @@ public void channelInactive(ChannelHandlerContext ctx) {
       }
     }
   }
+
+  static int parseOrderBookSize(Object[] args) {
+    if (args != null && args.length > 0) {
```

Comment:
can you move this logic outside to the methods that use it, we shouldn't have a method like this having to inspect an args[] object array.

also just maybe make it simpler since its one arg 
```
if (!(args != null && args.length > 0 && args[0] instanceof Number)) {
      LOG.error(
              "Order book size param type {} is invalid. Expected: {}. Default order book size has been used {}",
              obSizeParam.getClass().getName(),
              Number.class,
              ORDER_BOOK_SIZE_DEFAULT);
              return;
}
```

Reply:
this is the original implementation that was used to determine the levels, I've opted to leave that as is as much as possible since change is already quite large.